experiments/aws/parallel-cluster/cpu/experiment/results/32-node/stream/stream-1n-226-iter-1.err

```diff
@@ -0,0 +1,3 @@
+real 0.68
```

Comment:
Just to double check - this file pattern of error (.err) corresponds to the output file (.out) with a slightly different prefix (queue1-dy-*)?